pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -125,6 +126,9 @@ type ControllerConfigSpec struct {
 	// CAs
 	EtcdCAData []byte `json:"etcdCAData"`
 	RootCAData []byte `json:"rootCAData"`
+
+	// Admin
+	Admin installertypes.Admin `json:"admin"`
```

Comment:
i would like not import installer's type as there are more fields than we use.

Reply:
Will do!